support/modules/rootfs-image

```diff
@@ -5,10 +5,27 @@ set -ue
 STATE="$1"
 FILES="$2"
 
+if [ -n "$(which jq >/dev/null 2>&1)" ]; then
```

Comment:
Hmm, safeguarding for weird which behaviour?

Why not use `command`?

Reply:
> Hmm, safeguarding for weird which behaviour?

Something like:

```json
{
  "MyOwnStuff": {
    "RootfsPartA": "wrong-stuff",
    "RootfsPartB": "more-wrong-stuff"
  },
  "RootfsPartA": "/dev/sda2",
  "RootfsPartB": "/dev/sda3"
}
```

But as you can imagine, it's a pretty theoretical possibility. Why would anyone do that? 🙂  

> Why not use `command`?

`command` is not available in busybox.

Reply:
> command is not available in busybox.

hah, what? Really

Reply:
Which version of `sh` do they run?

Reply:
Busybox *is* `sh`. 🙂